digdag-core/src/main/java/io/digdag/core/notification/ShellNotificationSender.java

```diff
@@ -77,14 +82,27 @@ public void sendNotification(Notification notification)
             });
 
             boolean exited = process.waitFor(timeoutMs, MILLISECONDS);
+
+            BufferedReader br = new BufferedReader(new InputStreamReader(process.getInputStream()));
+            Stream<String> lines = br.lines();
+            StringBuilder sb = new StringBuilder();
+            lines.forEach(line -> {
+                sb.append(line);
+                sb.append(System.lineSeparator());
+            });
```

Comment:
If process create large error message, this may cause out of memory problem.
It may be better to set max size of log (e.x. 64k).